doc/nrf/glossary.rst

```diff
@@ -0,0 +1,426 @@
+
+Device Under Test (DUT)
+   A manufactured product undergoing testing.
+
+Docker
+   A set of software tools using OS-level virtualization to create and run applications and their dependencies in self-contained environments called containers.
+
+Docker container
+   A self-contained environment created by running a Docker container image in the Docker engine.
+
+Docker container image
+   A standard set of binary data that contains an application (or more than one) and all the required dependencies.
+   When run on the Docker engine, it creates a Docker container.
+
+Docker engine
+   The container runtime that runs Docker images.
+
+Domain Name System (DNS)
+   A hierarchical distributed naming system for computers, services, or any resource connected to the Internet or a private network.
+   It associates various information with domain names assigned to each of the participating entities.
+   Most prominently, it translates domain names, which can be easily memorized by humans, to the numerical IP addresses needed for computer services and devices worldwide.
+   The Domain Name System is an essential component of the functionality of most Internet services because it is the Internet's primary directory service.
+
+Downstream
+   The repository that is forked off an upstream.
+
+Firmware Over-the-Air (FOTA) update
+
+   A firmware update performed remotely over-the-air (OTA).
+
+Floating-Point Unit (FPU)
+   A part of a CPU specially designed to perform operations on floating-point numbers.
+
+Fork
+   A server-hosted copy of a repository (upstream) that intends to follow the changes made in the original repository as time goes by, while at the same time keeping some other changes unique to it.
+
+GitHub fork
+   A `GitHub fork`_ is a copy of a repository inside GitHub, that allows the user to create a Pull Request.
+
+General Packet Radio Services (GPRS)
+   Packet-based mobile data service for 2G and 3G mobile networks with data rates of 56-114 kbps/second and continuous connection to the Internet.
+
+General-Purpose Input/Output (GPIO)
+   A digital signal pin that can be used as input, output, or both.
+   It is uncommitted and controllable by the user at runtime.
+
+Generic Attribute Protocol (GATT)
+   "Generic Attribute Profile (GATT) is built on top of the Attribute Protocol (ATT) and establishes common operations and a framework for the data transported and stored by the Attribute Protocol.” Bluetooth Core Specification, Version 5.1, Vol 1, Part A, Section 6.4
+
+Global Navigation Satellite System (GNSS)
+   A satellite navigation system with global coverage.
+   The system provides signals from space transmitting positioning and timing data to GNSS receivers, which use this data to determine location.
+
+Global Positioning System (GPS)
+   A satellite-based radio navigation system that provides its users with accurate location and time information over the globe.
+
+GPIOTE
+   General-Purpose Input/Output Tasks and Events
```

Comment:
This just opens the abbreviation. Do we have a definition?

Reply:
fixed